src/ir_Mitsubishi.cpp

```diff
@@ -292,6 +343,129 @@ void IRsend::sendMitsubishiAC(unsigned char data[], uint16_t nbytes,
 }
 #endif  // SEND_MITSUBISHI_AC
 
+#if DECODE_MITSUBISHI_AC
+// Decode the supplied Mitsubishi message.
+//
+// Args:
+//   results: Ptr to the data to decode and where to store the decode result.
+//   nbits:   Nr. of data bits to expect.
+//   strict:  Flag indicating if we should perform strict matching.
+// Returns:
+//   boolean: True if it can decode it, false if it can't.
+//
+// Status: ALPHA / Under development
+//
+// Ref: https://www.analysir.com/blog/2015/01/06/reverse-engineering-mitsubishi-ac-infrared-protocol/
+bool IRrecv::decodeMitsubishiAC(decode_results *results, uint16_t nbits,
+                              bool strict) {
+  if (results->rawlen < ((MITSUBISHI_AC_STATE_LENGTH * 8 * 2) + 2)) {
```

Comment:
MITSUBISHI_AC_STATE_LENGTH * 8 is MITSUBISHI_AC_BITS
Use that instead

Reply:
You are right, I can modify to `MITSUBISHI_AC_STATE_LENGTH * 2` (because mars and spaces).